### PR TITLE
IGVF-239 Updated post-differentiation-time-unit.

### PR DESCRIPTION
**Pull request by KAndreeva**
Updated post-differentiation-time-unit. Added schema upgrade 4 for differentiated tissue and differentiated cell.

### CODE REVIEW

File: src/igvfd/schemas/changelogs/differentiated_tissue.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ### Schema version 4
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/differentiated_tissue.md:3]: this should have been Schema 5

File: src/igvfd/schemas/changelogs/differentiated_cell.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 ### Schema version 4
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/differentiated_cell.md:3]: should have been schema version 5

File: src/igvfd/schemas/differentiated_cell.json [hunk 3]
```diff
@@ -57,7 +57,7 @@
             "title": "Post-differentiation time",
             "description": "The time that elapsed past the point when biosamples were differentiated.",
```
Comment (gabdank) [inline src/igvfd/schemas/differentiated_cell.json:58]: you forgot to change default schema version to be 5

File: src/igvfd/schemas/differentiated_tissue.json [hunk 4]
```diff
@@ -72,7 +72,7 @@
             "title": "Post-differentiation time",
             "description": "The time that elapsed past the point when biosamples were differentiated.",
```
Comment (gabdank) [inline src/igvfd/schemas/differentiated_tissue.json:73]: you forgot to change default schema version to be 5

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/schemas/changelogs/differentiated_tissue.md (hunk 1 above)
Comment (KAndreeva) [reply src/igvfd/schemas/changelogs/differentiated_tissue.md]: Done, missed from re-basing.

File: src/igvfd/schemas/changelogs/differentiated_cell.md (hunk 2 above)
Comment (KAndreeva) [reply src/igvfd/schemas/changelogs/differentiated_cell.md]: Done, missed from re-basing.

File: src/igvfd/schemas/differentiated_cell.json (hunk 3 above)
Comment (KAndreeva) [reply src/igvfd/schemas/differentiated_cell.json:58]: Done, missed from re-basing.

File: src/igvfd/schemas/differentiated_tissue.json (hunk 4 above)
Comment (KAndreeva) [reply src/igvfd/schemas/differentiated_tissue.json:73]: Done, missed from re-basing.

**Comment by KAndreeva** [comment]
Closing pull request due to rebasing. Changes will be done in a different branch.

**Pull request closed without merging**